[OV JS] Add note about usage in development of Electron applications (#25475)
openvinotoolkit/openvino: src/bindings/js/node/README.md

const { addon: ov } = require('openvino-node');
```

> ## Package usage in Electron applications

Reviewer: ```suggestion
## Usage in Electron applications
```

> To use this package in development of Electron applications on Windows, make sure that
> in your system installed [Build Tools for Visual Studio](https://aka.ms/vs/17/release/vs_BuildTools.exe)

Reviewer: ```suggestion
To use the package in development of Electron applications on Windows, make sure that
[Build Tools for Visual Studio](https://aka.ms/vs/17/release/vs_BuildTools.exe) are installed.
```
